cs464-group-project/src/pages/Players/IndividualPlayer.js

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import axios from 'axios';
+
+export function IndividualPlayer() {
+  const { id } = useParams();
+  const [playerData, setPlayerData] = useState(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await axios.get(
+          `https://www.thesportsdb.com/api/v1/json/3/lookupplayer.php?id=${id}`
```

Comment:
Place the api calls in the ApiRequest page, and use the calls in the sportsdb library

Reply:
Changed api call to use sportdb library